nncf/quantization/algorithms/min_max/onnx_backend.py

```diff
@@ -96,6 +96,10 @@ def hw_config(self) -> HWConfig:
     def quant_trait_op_dict(self) -> Dict[int, OperatorMetatype]:
         return DEFAULT_ONNX_QUANT_TRAIT_TO_OP_DICT
 
+    @staticmethod
+    def get_start_nodes_for_activation_path_tracing(nncf_graph: NNCFGraph) -> List[OperatorMetatype]:
+        return nncf_graph.get_input_nodes()
```

Comment:
`nncf_graph.get_input_nodes()` is not the list of `OperatorMetatype` here and below.

Reply:
My bad, thanks! Done